blocks/library/latest-posts/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlock } from '../../api';
+import { getLatestPosts } from './data.js';
+
+
+registerBlock( 'core/latestposts', {
+	title: wp.i18n.__( 'Latest Posts' ),
+
+	icon: 'list-view',
+
+	category: 'rest-api',
+
+	edit: class extends wp.element.Component {
+		constructor() {
+			super( ...arguments );
+
+			this.state = {
+				latestPosts: []
+			};
+
+			let { poststoshow } = this.props.attributes;
+
+			if ( ! poststoshow ) {
+				poststoshow = 5;
+
+				this.props.setAttributes( {
+					poststoshow
+				} );
```

Comment:
> I'm happy to put the SSR endpoint into its own PR, can start on it now if it's wanted :)

No rush, I think I'll leave the SSR part as it is now and we can improve it in a future PR. :)